deploy/helm/virtink/templates/virt-controller/manifests.yaml

```diff
@@ -31,15 +33,17 @@ apiVersion: admissionregistration.k8s.io/v1
 kind: ValidatingWebhookConfiguration
 metadata:
   creationTimestamp: null
-  name: validating-webhook-configuration
+  name: virt-controller
```

Comment:
But by supporting this, I must use a more hacker way like

```bash
annotations="  annotations:
    cert-manager.io/inject-ca-from: {{ .Release.Namespace }}/virt-controller-cert"
mutating_replace="  name: virtink-mutating-webhook-configuration\n$annotations"
validating_replace="  name: virtink-validating-webhook-configuration\n$annotations"

# Replace newlines with literal \n, replace \ -> \/ for sed replace below
mutating_replace="$(echo "${mutating_replace//$'\n'/\\n}" | sed "s/\//\\\\\//g")"
validating_replace="$(echo "${validating_replace//$'\n'/\\n}" | sed "s/\//\\\\\//g")"
```

instead of using more simpler codes

```bash
replace="  name: virt-controller
  annotations:
    cert-manager.io/inject-ca-from: {{ .Release.Namespace }}/virt-controller-cert"
```

Reply:
yes, `sed` is not good for maintenance, how about using `kustomize` during generate.  generate.sh generate the intermediate webhook manifests.yaml in to `hack/webhook`, and execute `kustomize build` to render the final manifests.